.github/workflows/build_and_test.yml

```diff
@@ -743,8 +734,6 @@ jobs:
         Rscript -e "install.packages(c('devtools', 'testthat', 'knitr', 'rmarkdown', 'markdown', 'e1071', 'roxygen2', 'ggplot2', 'mvtnorm', 'statmod'), repos='https://cloud.r-project.org/')"
         Rscript -e "devtools::install_version('pkgdown', version='2.0.1', repos='https://cloud.r-project.org')"
         Rscript -e "devtools::install_version('preferably', version='0.4', repos='https://cloud.r-project.org')"
-    - name: Install dependencies for documentation generation
```

Comment:
this step is also used in 3.3/3.4/3.5, so move to `Install R linter dependencies for branch-3.3, branch-3.4, branch-3.5`